apps/nextjs-app/src/features/app/blocks/view/list/ViewListItem.tsx

```diff
@@ -135,19 +129,25 @@ export const ViewListItem: React.FC<IProps> = ({ view, removable, isActive }) =>
         }
         navigateHandler();
       }}
+      onContextMenu={() => showViewMenu && setOpen(true)}
```

Comment:
Here we need to consider the interaction in touch devices.

Reply:
@Sky-FE Right-click is just a new, alternative way to open this menu. The old method of double-clicking haven't got deleted and is compatible with mobile devices.

Reply:
Oh, i misunderstood.